src/caselawclient/models/search_results.py

```diff
@@ -1,52 +0,0 @@
-from typing import List
-
-from lxml import etree
-
-
-class SearchResults:
```

Comment:
This gets added back in https://github.com/nationalarchives/ds-caselaw-custom-api-client/pull/223/files#diff-05cdf70bd9924bdbcf34f282af03471503bbfc4db7634f87d89ce246acabda62